cdap-ui/app/hydrator/leftpanel.less

```diff
@@ -309,9 +309,9 @@ body.theme-cdap.state-hydrator-create {
         }
         div.side-panel.top {
           .hydrator-metadata {
-            width: ~"calc(100vw - 755px)";
-            width: ~"-moz-calc(100vw - 755px)";
-            width: ~"-webkit-calc(100vw - 755px)";
+            width: ~"calc(100vw - 805px)";
```

Comment:
This is a problem when the sidepanel is minimized. 
<img width="1680" alt="screen shot 2017-05-24 at 7 26 06 pm" src="https://cloud.githubusercontent.com/assets/1452845/26433336/e2f1ca12-40b6-11e7-8124-54e40ca4b72c.png">